subsys/bluetooth/shell/gatt.c

```diff
@@ -214,6 +213,7 @@ static uint8_t discover_func(struct bt_conn *conn,
 static int cmd_discover(const struct shell *sh, size_t argc, char *argv[])
 {
 	int err;
+	static struct bt_uuid_16 uuid;
```

Comment:
It consumes two more memory sizes for UUID just due to the naming issues. Looks wasteful.